lib/vagrant-service-manager/plugin_util.rb

```diff
@@ -111,6 +111,7 @@ def self.execute_and_exit_on_fail(machine, ui, command)
 
       def self.execute_once(machine, ui, command)
         machine.communicate.sudo(command) do |_, data|
+          next if data.chomp.empty?
```

Comment:
Created https://github.com/mitchellh/vagrant/issues/8259 expecting any reply from vagrant folks. I will debug it in my free time considering other priorities.